src/login/login.component.tsx

```diff
@@ -51,22 +45,9 @@ export default function Login(props: LoginProps) {
           width: 300px;
         `}
       >
-        <div
-          css={css`
-            margin: 20px;
-          `}
-        >
-          <img
-            src={getLogoURL()}
-            alt="openmrs-logo"
-            css={css`
-              max-width: 85%;
-              max-height: 100%;
-              display: block;
-              margin: auto;
-            `}
-          />
-        </div>
+        <svg role="img">
+          <use xlinkHref="#omrs-logo-full-color" />
```

Comment:
A css-only way of vending svgs from the styleguide to other modules that want to use them. See https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--logos

Reply:
nice. lgtm